data/quarg/quarg missions.txt

```diff
@@ -259,3 +259,44 @@ mission "Ask Quarg About Coalition Early"
 			label end
 			`	Before you can ask it anything else, the Quarg bids you farewell, and walks away.`
 				decline
+
+
+
+mission "Quarg Pug Arfecta Warning"
+	landing
+	to offer
+		has "First Contact: Quarg: offered"
+		has "flagship model: Pug Arfecta"
+	source
+		attributes "quarg"
+		attributes "station"
```

Comment:
why only on stations?

Reply:
At first I just copied the "Ask Quarg About Coalition Early" mission to get started.

I looked at that  source condition and thought maybe I should remove it, but then I thought there was more narrative weight if they warned you about the power of the ship, a ship that was most likely used to destroy one of their rings, on one of their rings.

Reply:
There is a `"ringworld"` attribute for that, which differs from stations

Reply:
Gotcha will make that change. I guess the station in human space doesn't count as a ring world because the ring isn't done yet.